README.md

```diff
@@ -14,6 +14,19 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
+
+⚠️ **WARNING** ⚠️
+================
+This extension has several significant limitations:
+- No location tracking support, making error messages less helpful
+- Uses an unmaintained library as its core dependency
+
+**Consider using [Mason](https://github.com/maveniverse/mason) instead, which provides a more robust and maintained alternative.**
+
+📢 **ARCHIVED REPOSITORY** 📢
+==========================
+This repository has been archived and is no longer maintained. Please use [Mason](https://github.com/maveniverse/mason) instead.
```

Comment:
I'm afraid that we are trying to move out from ASF for too many Maven projects ...

I'm feel we trying to build Maven ecosystem outside ASF ...

Reply:
The folks may freely choose where they invest their time an effort. I think it is fine to honestly say that they do not plan to work on this repo anymore. OTOH if there is anybody else wanting to push this repo forward, then it would also be fine to un-archive it again.

Reply:
Also there was a thread on ML where some expressed opinion to not keep these "experiments" in ASF Maven as that may show false pictures of how we want to maintain these. Having it in Maveniverse is not like we don't (is not opposite) but we are not saying "it's official" either. And as Peter say, whoever needs it may fork it. Altough new work by Guillaume is superior to this one, so am unsure is unarchiving this one really feasible.

Reply:
For ref: https://lists.apache.org/thread/m55h1brfq9xqkw6xjd7ndgq1lqryt9xk

Reply:
> I'm afraid that we are trying to move out from ASF for too many Maven projects ...
> 
> I'm feel we trying to build Maven ecosystem outside ASF ...

Well, if there was no pushback from the community, we could build an ecosystem inside the ASF.  I'm *all for* having all those tools inside Maven, but I don't want to have to fight each time we're experimenting with something...

Reply:
I hope no one was offended by my comment ... I only think too loudly and maybe to optimistic sometime 😄

Reply:
No, you are completely right. But, others might be as well. So I think having a "lab outpost" is useful. Quick prototyping, "quick moving", and once we (mvn community) think is okay to "bring it in", bring it in.